client/ts/src/wrapperObj.ts

```diff
@@ -41,8 +41,9 @@ export interface MarketInfoRaw {
   traderIndex: number;
   baseBalanceAtoms: bignum;
   quoteBalanceAtoms: bignum;
+  quoteVolumeAtoms: bignum;
```

Comment:
I got a pull request open to fix all this

Reply:
ok cool. but for now we leave it and merge your changes in, right?

Reply:
If yours works and all the tests pass, then you got there first, so I will deal with any merges later after yours is on main.